ai/src/algorithms/sacd/sacd.json

```diff
@@ -0,0 +1,5 @@
+{
+    "algorithm_id": "sacd",
+    "name": "Soft Actor Critic Discrete",
+    "docs_link": "https://docs.spiceai.org/deep-learning-ai/sac/"
```

Comment:
Are you planning to make this docs change today? Otherwise I'll create an issue to track it.

Reply:
Let's make an issue indeed, this is probably be made latter